RoboJackets/igvc-software
# Back Circle Service

# Description
Adds a layer to the costmap that prevents the robot from pathing backwards.

Fixes #749 

## Test Case 1
1. Start up gazebo simulation
2. Launch navigation with read_from_file:=false
3. Launch rviz and set a navigation waypoint
4. Set another one later

Expectation: The back circle should move with the robot and always generate behind it when setting waypoints.

## Test Case 2
1. Start up gazebo simulation
2. Launch navigation with read_form_file:=true

Expectation: The back circle will be generated behind the robot once it reads from the waypoint file.

# Self Checklist
